src/context.rs

```diff
@@ -81,7 +81,7 @@ pub trait UsbContext: Clone + Sized {
         if handle.is_null() {
             None
         } else {
-            Some(unsafe { device_handle::from_libusb(self.clone(), handle) })
+            Some(unsafe { DeviceHandle::from_libusb(self.clone(), std::ptr::NonNull::new_unchecked(handle)) })
```

Comment:
small nitpick 
```suggestion
            Some(unsafe { DeviceHandle::from_libusb(self.clone(), NonNull::new(handle)?) })
```
And we can remove `if handle.is_null()`

Reply:
How about? ```        let ptr = std::ptr::NonNull::new(handle)?;
        Some(unsafe { DeviceHandle::from_libusb(self.clone(), ptr) })
```